### DISCUSSION TITLE
Multi-ranger deck and motion capture system

### DISCUSSION THREAD

**Discussion by MINJEEgi** [USC-ACTLab/crazyswarm#750] [Q&A]
Hello, 
We are planning to map a space using multiple crazyflies with the multi-ranger deck through a motion capture system. Before conducting this experiment, we have a question and would like to reach out. We are wondering if it's possible to simultaneously receive values from multiple multi-ranger decks when using them in the motion capture system.

**Comment by whoenig** [comment]
In principle you can log from multiple crazyflies, but the time won't be synchronized and you are quite limited in terms of frequency. If you want to use offline mapping, a better way would be to store the data on a uSD card, for each Crazyflie.

**Reply by MINJEEgi** [reply]
thanks for your replay. I have a few more questions. 

1. Can we achieve precise control using optitrack tracking and receive data from the multi-ranger sensors (front, back, left, right)? We need two types of data: 'timestamp' and 'crazyflie 6-dof pose,' as well as corresponding 'timestamp' and 'distance measurements for front, back, left, and right.' 

2. Does 'time won't be synchronized' mean that each drone operates on a different time reference point during their interaction? or related to the synchronization between the 6-dof pose and the multi-ranger values?
